chart/skywalking/values.yaml

```diff
@@ -97,6 +97,7 @@ ui:
     # clusterIP: None
     externalPort: 80
     internalPort: 8080
+    # nodePort: 30080
```

Comment:
30080 is meaningless, it is recommended to use the default value range (30000-32767) instead @chengshiwen

Reply:
30080 is just a sample. In some k8s, only operation people have the rights to to allocate node ports, and random node port is not allowed. It would be a better option to allow node port to be specified.

Reply:
My mean is that the example does not need a specific port, it can be a default port range, or prompt the user in the readme file @chengshiwen